src/bindings/config.ts

```diff
@@ -84,6 +91,14 @@ export const loadConfig = async (context: Context): Promise<BotConfig> => {
     wallet: {
       registerWalletWithVerification: registerWalletWithVerification,
     },
+    time: {
+      DEFAULT_TIME_RANGE_FOR_MAX_ISSUE: process.env.DEFAULT_TIME_RANGE_FOR_MAX_ISSUE
```

Comment:
the variable names should be in snakeCase and I think it's better to put them in unassign object because it's related to unassigning